Orange/classification/tree.py

```diff
@@ -1,20 +1,254 @@
+"""Tree inducers: SKL and Orange's own inducer"""
+
+import numpy as np
+import bottleneck as bn
 import sklearn.tree as skl_tree
+from Orange.statistics import distribution, contingency
 
-from Orange.base import Tree
+from Orange.base import Tree, Model, Learner
```

Comment:
Did you want to extend Orange.base.Learner, thus supporting both classification and regression within the same class, or Orange.classification.Learner, which can perform only classification?